packages/invoices/src/OutboundShipment/ListView/Toolbar.tsx

```diff
@@ -12,7 +12,8 @@ import { useOutbound, OutboundRowFragment } from '../api';
 
 export const Toolbar: FC<{
   filter: FilterController;
-}> = ({ filter }) => {
+  updateFilter: (key: string, value: string) => void;
```

Comment:
it should be possible to use the method on the `filter` object - I'd look at removing the params from the list hook; they were only put in there to fix the issue of having multiple hooks call different params instances. Now that we have a global state, that can be undone and the filter can be the `useUrlQuery` version only, which is then called by `Toolbar` and `columns`